src/SQLite.cs

```diff
@@ -338,6 +338,29 @@ public TableMapping GetMapping<T> ()
 			return GetMapping (typeof (T));
 		}
 
+		/// <summary>
+		/// Adds a mapping to be used for indexes and naming for the given type.
+		/// </summary>
+		/// <param name="tableMapping">The table mapping.</param>
+		internal void AddMapping(TableMapping tableMapping)
```

Comment:
I don't like this. Instead, we'll have the builder create a mapping and have CreateTable accept it.